goal_src/engine/ps2/pad.gc

```diff
@@ -172,6 +172,38 @@
     )
   )
 
+(defun v1-analog-input ((in int) (offset float) (center-val float) (max-val float) (out-range float))
+  "Same as analog-input but respects First-Person Vertical camera control setting."
+  (*
+    (if (and PC_PORT (-> *pc-settings* first-camera-vflip?)) 1.0 -1.0)
+    (analog-input in offset center-val max-val out-range)
+  )
+)
+
+(defun h1-analog-input ((in int) (offset float) (center-val float) (max-val float) (out-range float))
+  "Same as analog-input but respects First-Person Horizontal camera control setting."
+  (*
+    (if (and PC_PORT (-> *pc-settings* first-camera-hflip?)) 1.0 -1.0)
+    (analog-input in offset center-val max-val out-range)
+  )
+)
+
+(defun v3-analog-input ((in int) (offset float) (center-val float) (max-val float) (out-range float))
+  "Same as analog-input but respects Third-Person Vertical camera control setting."
+  (*
+    (if (and PC_PORT (-> *pc-settings* third-camera-vflip?)) 1.0 -1.0)
+    (analog-input in offset center-val max-val out-range)
+  )
+)
+
+(defun h3-analog-input ((in int) (offset float) (center-val float) (max-val float) (out-range float))
+  "Same as analog-input but respects Third-Person Horizontal camera control setting."
+  (*
+    (if (and PC_PORT (-> *pc-settings* third-camera-hflip?)) 1.0 -1.0)
+    (analog-input in offset center-val max-val out-range)
+  )
+)
+
```

Comment:
You can probably give these a more descriptive name like `analog-input-horizontal-third` and make them a macro instead of a whole since it's a small piece of code and the arguments are used only once, and it saves us 4 symbols. It's a very simple replacement macro so you just add a backquote ` at the start and a comma , before each of the arguments.